src/authenticators/mod.rs

```diff
@@ -12,6 +12,7 @@
 //! Currently only a simple Direct Authenticator component is implemented.
 
 pub mod direct_authenticator;
+pub mod uds_authenticator;
```

Comment:
Is it useful to perhaps wrap this authenticator in a Rust/Cargo feature so that it can be selected for at build time? (Actually, looking at the rest of the change, perhaps you've already done that? But there's an indication that it's only temporary. I actually think there is value in having this be a selectable thing even longer term, because it's useful for people to be able to compile down a Parsec executable with only the precise pieces that their deployment needs. Providers and authenticators are the kinds of things that we might want to snip out, because they can be heavyweight, especially in terms of the dependencies that get dragged in with them).

Reply:
Yes, good idea. Will do this. 👍